lua/core/mappings.lua

```diff
@@ -193,6 +193,58 @@ if is_available "toggleterm.nvim" then
   maps.t["<C-'>"] = maps.n["<F7>"]
 end
 
+if is_available "nvim-dap" then
```

Comment:
it would be interesting to see if `mappings.lua` could more closer to the config for each plugin. way out of the scope of this PR ofc